bundles/test/org.eclipse.smarthome.test/src/test/java/org/eclipse/smarthome/test/java/JavaTestTest.java

```diff
@@ -52,15 +52,15 @@ public void waitForAssertShouldRunAfterLastCall_whenAssertionSucceeds() {
     public void waitForAssertShouldRunAfterLastCall_whenAssertionFails() {
         Runnable allwaysFailAssertion = mock(Runnable.class);
         doAnswer(invocation -> {
-            throw new NullPointerException();
+            throw new AssertionError();
         }).when(allwaysFailAssertion).run();
 
         Runnable afterLastCall = mock(Runnable.class);
 
         try {
             javaTest.waitForAssert(allwaysFailAssertion, null, afterLastCall, 10, 5);
             fail("NPE from mock is expected during last execution.");
```

Comment:
please also adapt the fail message

Reply:
@tanyatanyatanya please just update this minor thing. We are good to go then. Thanks!

Reply:
@htreu I will update it but the Travis build still fails with:

Failed to execute goal org.eclipse.tycho:tycho-compiler-plugin:1.1.0:compile (default-compile) on project org.eclipse.smarthome.test: Compilation failure: Compilation failure: 
/home/travis/build/eclipse/smarthome/bundles/test/org.eclipse.smarthome.test/src/main/java/org/eclipse/smarthome/test/java/JavaOSGiTest.java:[26] 
import org.eclipse.smarthome.test.OSGiTest;

The import org.eclipse.smarthome.test.OSGiTest cannot be resolved`

@maggu2810 @htreu Can you help on that? I will appreciate it. Thanks!